src/main/java/com/google/api/codegen/transformer/ruby/RubyPackageMetadataTransformer.java

```diff
@@ -70,6 +89,58 @@ private ViewModel generateGemspecView(Model model, RubyPackageMetadataNamer name
         .build();
   }
 
+  private ViewModel generateReadmeView(
+      Model model, GapicProductConfig productConfig, RubyPackageMetadataNamer namer) {
+    List<ApiMethodView> exampleMethods = generateExampleMethods(model, productConfig);
+    return metadataTransformer
+        .generateMetadataView(
+            packageConfig, model, README_FILE, README_OUTPUT_FILE, TargetLanguage.RUBY)
+        .identifier(namer.getMetadataIdentifier())
+        .fileHeader(
+            fileHeaderTransformer.generateFileHeader(
+                productConfig,
+                ImportSectionView.newBuilder().build(),
+                new RubySurfaceNamer(productConfig.getPackageName())))
+        .developmentStatus(
+            namer.getReleaseAnnotation(packageConfig.releaseLevel(TargetLanguage.RUBY)))
+        .exampleMethods(exampleMethods)
+        .build();
+  }
+
```

Comment:
I guess we should indicate currently the example method is based on 'smoke test' method in the comment.